documentation/docs/01-getting-started/01-introduction.md

```diff
@@ -2,19 +2,13 @@
 title: Introduction
 ---
 
-Welcome to Svelte docs. These are for folks who are already familiar with Svelte and want to learn more about the details of how it works.
+Welcome to the Svelte reference documentation! This is intended as a resource for people who are already have some familiarity with Svelte and want to learn more about the details of how it works.
 
-If that's not you (yet), you may prefer to visit the [interactive tutorial](/tutorial) or the [examples](/examples) before consulting this reference.
-
-Don't be shy about asking for help in the [Discord chatroom](https://svelte.dev/chat).
-
-Using an older version of Svelte? Have a look at the [v2 docs](https://v2.svelte.dev).
+If that's not you (yet), you may prefer to visit the [interactive tutorial](https://learn.svelte.dev) or the [examples](/examples) before consulting this reference. You can try Svelte online using the [REPL](/repl) or (for a more fully-featured environment) [StackBlitz](https://sveltekit.new).
```

Comment:
Not sure about the parens here. Maybe should be an emdash?
```suggestion
If that's not you (yet), you may prefer to visit the [interactive tutorial](https://learn.svelte.dev) or the [examples](/examples) before consulting this reference. You can try Svelte online using the [REPL](/repl) or—for a more fully-featured environment—[StackBlitz](https://sveltekit.new).
```

Or maybe just rewrite?
```suggestion
If that's not you (yet), you may prefer to visit the [interactive tutorial](https://learn.svelte.dev) or the [examples](/examples) before consulting this reference. You can try Svelte online using the [REPL](/repl). Alternatively, if you'd like a more fully-featured environment, you can try Svelte on [StackBlitz](https://sveltekit.new).
```